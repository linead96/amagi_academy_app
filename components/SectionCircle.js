import React from 'react';
import {View} from 'react-native'

export default class SectionCircle extends React.Component {
  render() {
    return (
        <View
        style={{ 
            backgroundColor: this.props.color, 
            width: 90,
            height: 90,
            borderRadius: 45,
            margin: 30,    
        }}
        />
    );
  }
}