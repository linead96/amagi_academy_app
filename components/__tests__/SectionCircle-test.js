import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import SectionCircle from '../SectionCircle';

test('renders touchablehighlight component which reflects the passed props color, in this test the color is red', () => {
  const tree = renderer.create(
    <SectionCircle
      onPressButton={jest.fn()}
      color="red"
    />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
