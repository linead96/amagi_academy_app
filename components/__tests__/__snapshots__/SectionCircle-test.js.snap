// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders touchablehighlight component which reflects the passed props color, in this test the color is red 1`] = `
<View
  accessible={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "red",
      "borderRadius": 45,
      "height": 90,
      "margin": 30,
      "transform": Array [
        Object {
          "scale": 1,
        },
      ],
      "width": 90,
    }
  }
/>
`;
